.github/workflows/build_and_test.yml

```diff
@@ -75,10 +75,12 @@ jobs:
     # paho.mqtt requires openssl and OPENSSL_DIR on Windows
     - name: Install OpenSSL (Windows)
       if: matrix.os == 'windows-latest'
-      run: |
```

Comment:
The `|` pipe is necessary because there are multiple commands running in this step

Reply:
Just noticed that, thanks